opsdroid/connector/matrix/connector.py

```diff
@@ -413,45 +435,46 @@ async def _send_room_creation(self, creation_event):
     @register_event(events.RoomName)
     @ensure_room_id_and_send
     async def _send_room_name_set(self, name_event):
-        return await self.connection.set_room_name(name_event.target, name_event.name)
+        return await self.connection.room_put_state(
+            name_event.target, "m.room.name", name_event.name
+        )
 
     @register_event(events.RoomAddress)
     @ensure_room_id_and_send
     async def _send_room_address(self, address_event):
-        try:
-            return await self.connection.set_room_alias(
-                address_event.target, address_event.address
-            )
-        except MatrixRequestError as err:
-            if err.code == 409:
-                _LOGGER.warning(
-                    f"A room with the alias {address_event.address} already exists."
-                )
+        return await self.connection.room_put_state(
+            address_event.target, "m.room.aliases", address_event.address
+        )
+        # except MatrixRequestError as err:
+        #    if err.code == 409:
+        #        _LOGGER.warning(
+        #            f"A room with the alias {address_event.address} already exists."
+        #        )
```

Comment:
Was there a particular reason for removing this warning?

Reply:
Nope, added them in again